infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationManager.java

```diff
@@ -203,6 +203,12 @@ public void processStandbyChange(TopologyDescriptor newConfig) {
                 startLogReplicationRuntime(clusterInfo);
             }
         }
+
+        //Update intersections' router
```

Comment:
This comment would probably provide more info if it indicates that the connection id's could've changed for existing standby cluster's, updating the routers will re-establish the connection to the correct endpoints (something in that line).. otherwise this comment doesn't say much.

Reply:
Done!